aiobotocore/endpoint.py

```diff
@@ -199,7 +190,6 @@ def _request(self, method, url, headers, data):
                                                     headers=headers_,
                                                     data=data,
                                                     proxy=proxy,
-                                                    timeout=self._read_timeout,
```

Comment:
@achimnol  this is interesting, should we be removing it or setting it to `None` given we're setting the read + conn timeouts on the session now? I think the default was 5 minutes. Along those lines I've opened issue: https://github.com/aio-libs/aiohttp/issues/1757.